### PR TITLE
 fix issue #318 &  Quick hack: Get link without usernames 

### PR DESCRIPTION
**Pull request by riskey95**

### CODE REVIEW

File: lua/main.lua [hunk 1]
```diff
@@ -141,8 +141,7 @@ Unfortunately I can't work in normal groups. If you need me, please ask the crea
 
 			-- log this event
 			if config.bot_settings.stream_commands then
-				print(string.format('%s[%s]%s Bot was added to a normal group %s%s [%d] -> [%d]',
-					clr.blue, os.date('%X'), clr.yellow, clr.reset, msg.from.first_name, msg.from.id, msg.chat.id))
+				print(clr.blue..'['..os.date('%X')..']'..clr.yellow..' Bot was added to a normal group'..clr.reset..''..msg.from.first_name..' ['..msg.from.id..'] -> ['..msg.chat.id..']')
```
Comment (NotAFile) [inline lua/main.lua]: not a big fan of this. I'd much prefer actually checking the relevant keys.

Comment (riskey95) [reply lua/main.lua]: ``lua: ./lua/main.lua:144: bad argument #2 to 'format' (string expected, got table)
stack traceback:
	[C]: in function 'format'
	./lua/main.lua:144: in function <./lua/main.lua:61>
	(tail call): ?
	polling.lua:47: in main chunk
	[C]: ?``

Comment (riskey95) [reply lua/main.lua]: why don't you merge this pull request?

**Comment by yangm97** [comment]
Well, I’m going to ignore the log thing because:
1. I want to get rid of this error asap;
2. I hope to introduce a log module.

But please don’t add unrelated changes on your next PRs. Thanks.

**Pull request merged**